documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,158 @@
+# CAMARA Security and Interoperability Profile
+
+## Table of Contents
+<!-- TOC -->
+* Introduction
+* Audience
+* Conventions
+* General Considerations
+    * Transport Security
+* OIDC Authorization Code Flow
+    * Cross-Site Request Forgery Protection
+* Client-Initiated Backchannel Authentication Flow
+    * Authentication Request
+* Client Credentials Flow
+    * Access Token Request
+* The Scope Parameter
+* Client Authentication
+* References
+<!-- TOC -->
+
+
+## Introduction
```

Comment:
@sfnuser Please propose some intro text. Is there similar text in https://github.com/camaraproject/IdentityAndConsentManagement/pull/113 you are happy with regarding future enhancements and backwards compatibility?

Reply:
@AxelNennker Thanks for looking into it. I asked the same in https://github.com/camaraproject/IdentityAndConsentManagement/pull/113 but was not answered - lost in commit overwrites I guess.

Honestly, I do not have a clarity on the scope and hence I posted the question in case I am missing something. This project has a release cycle and will OIDC profile be part of the release? As OIDC profile governs all CAMARA sub-projects, how will the API specifications refer to the OIDC profile? For e.g. Will the API specification yaml have a line that `this API confirms to ICM project release v0.3.0`? This will be helpful to track AZ feature sets that are supported in an API. Maybe we could start with this approach and if and when a need arise we can think of OIDC profile spec alone version controlled separately within the project. If there is consensus, we could add this text in the introduction section. Sub-project members will participate actively as the scope is now clear. What do you think?

I think none of the CAMARA API versions are backwards compatible yet. If we have an agreement on the above point, it would be clear on how APIs approach OIDC profile when deciding API versioning.

Reply:
I assume that is for the TSC to decide. Many requirements important to Camara implementers and clients are in commonalities, how is this done with commonalities?
@sfnuser Please create a new separate issue about release cycles etc and resolve this conversation

Reply:
@hdamker @AxelNennker OK. Fine with me 👍🏼